components/calendar/src/hebrew.rs

```diff
@@ -116,14 +116,35 @@ impl Calendar for Hebrew {
         _month_code: types::MonthCode,
         _day: u8,
     ) -> Result<Self::DateInner, CalendarError> {
-        // let year = if era.0 == tinystr!(16, "hebrew") {
-        //     year
-        // } else {
-        //     return Err(CalendarError::UnknownEra(era.0, self.debug_name()));
-        // };
-
-        // ArithmeticDate::new_from_codes(self, year, month_code, day).map(HebrewDateInner)
-        todo!("#3789")
+        let year = if _era.0 == tinystr!(16, "hebrew") {
```

Comment:
nit: we should remove the underscores from `_year` and `_era` if we're using them

Reply:
done